scripts/build_natura_raster.py

```diff
@@ -151,12 +151,7 @@ def unify_protected_shape_areas(inputs, natura_crs, out_logging):
         shp = gpd.read_file(i).to_crs(natura_crs)
         total_files += 1
         try:
-            shp.geometry = shp.apply(
```

Comment:
It looks like we may need to slightly revise the docstring. In particular [this line](https://github.com/davide-f/pypsa-earth/blob/6ecd9a903f8bbe678ef0b9157efb68d154d06158/scripts/build_natura_raster.py#L26) could be revised to something like:
`- ``data/raw/protected_areas/current/WDPA_Nov2023_Public_shp.zip``: `WDPA <https://en.wikipedia.org/wiki/Natura_2000>`_ World Database for Protected Areas.`

Reply:
Agree, revising